articles/welcome.md

```diff
@@ -40,7 +40,9 @@ Watch this [video](https://www.microsoft.com/en-us/videoplayer/embed/RE2JOJf) to
 
 ## Next steps
 
-The tutorials and articles above are meant to guide both the coder and non-coder through an introduction to quantum computing and Q#.  When you’re ready to continue to learn how to program quantum computers with Q#, here are the next steps:
+The Getting Started guides and articles above are meant to guide both the coder and non-coder through an introduction to quantum computing and Q#.  When you’re ready to continue to learn how to program quantum computers with Q#, here are the next steps:
```

Comment:
in response to my comment above, I think this would be sufficient rather than mentioning for every quickstart/tutorial that you don't need to be a coder to learn something from the tutorials. Maybe we pull some of this language into the top of section so that we only say it once vs for each quickstart.